common.go

```diff
@@ -47,7 +47,7 @@ type RepublishConditionFunc func(job *Job) bool
 // RepublishConditions alias of a list RepublishConditionFunc
 type RepublishConditions []RepublishConditionFunc
 
-// Comply checks if the Job fit in any of the defined conditions.
+// Comply checks if the Job fits any of the defined conditions.
```

Comment:
fits? maybe matches?

Reply:
yeah that's better, I'll push the change